tutorials/11-probabilistic-pca/11_probabilistic-pca.jmd

```diff
@@ -5,53 +5,53 @@ weave_options:
+For example, we have a data matrix $\mathbf{X} \in \mathbb{R}^{N \times D}$, and we would like to extract $k << D$ principal components which captures most of the variation information from the original matrix but live in low dimensions. The goal is to understand $\mathbf{X}$ through a lower dimensional subspace (e.g. two-dimensional subspace for visualisation convenience) spanned by the principal components.
 
-In order to do this, we need to find the principal directions where most of the variations of $\mathbf{A}$ aligns. This is can be achieved by decomposing $\mathbf{A}$. Traditionally, this is implemented via SVD which provides a robust and accurate computational framework for decomposing feature matrix, particularly for large datasets(see an illustration [here](https://intoli.com/blog/pca-and-svd/)):
+In order to project the original data matrix into low dimensions, we need to find the principal directions where most of the variations of $\mathbf{X}$ lie in. Traditionally, this is implemented via [singular value decomposition (SVD)](https://en.wikipedia.org/wiki/Singular_value_decomposition) which provides a robust and accurate computational framework for decomposing matrix into products of rotation-scaling-rotation matrices, particularly for large datasets(see an illustration [here](https://intoli.com/blog/pca-and-svd/)):
 
 $$
-\mathbf{A} \in \mathbb{R}^{m \times n} =  \mathbf{U} \in \mathbb{R}^{m \times r} \boldsymbol{\Sigma} \in \mathbb{R}^{r \times r}  \mathbf{V}^T \in \mathbb{R}^{r \times n}
+\mathbf{X}_{N \times D} =  \mathbf{U}_{N \times r} \times \boldsymbol{\Sigma}_{r \times r}  \times  \mathbf{V}^T_{r \times D}
 $$
 
-where $\Sigma_{d \times d}$ contains only $r$ (where $r<=min(m,n)$ is the rank of $A$) non-zero singular values. If we pad $\Sigma$ with zeros and add arbitrary orthonormal columns to $U$ and $V$, we obtain the more compact  orm [1]:
+where $\Sigma_{r \times r}$ contains only $r$ (where $r<=min(N,D)$ is the rank of $A$) non-zero singular values of $\mathbf{X}$. If we pad $\Sigma$ with zeros and add arbitrary orthonormal columns to $U$ and $V$, we obtain the more compact form [1]:
 
 $$
-A_{m \times n} =  U_{m \times m} \Sigma_{m \times n}  V_{n \times n}^T
+\mathbf{X}_{N \times D} = \mathbf{U}_{N \times N} \mathbf{\Sigma}_{N \times D} \mathbf{V}_{D \times D}^T
 $$
 
-where $U$ and $V$ are unitary matrices (i.e. with orthonormal columns). Such decomposition always exists for any complex matrix. columns of $V$ are the principal directions/axes. Percentage of variations explained can be calculated using the ratios of singular values.
+where $U$ and $V$ are unitary matrices (i.e. with orthonormal columns). Such decomposition always exists for any matrix. Columns of $V$ are the principal directions/axes. Percentage of variations explained can be calculated using the ratios of singular values [3].
```

Comment:
```suggestion
where $\mathbf{U}$ and $\mathbf{V}$ are unitary matrices (i.e. with orthonormal columns). Such decomposition always exists for any matrix. Columns of $\mathbf{V}$ are the principal directions/axes. Percentage of variations explained can be calculated using the ratios of singular values [3].
```

Reply:
yes, it's standard to use bold capital for matrix as you suggested.